core/ecschema-metadata/src/Metadata/Constant.ts

```diff
@@ -51,19 +55,20 @@ export class Constant extends SchemaItem {
     if (this.phenomenon !== undefined)
       schemaJson.phenomenon = this.phenomenon.fullName;
     schemaJson.definition = this.definition;
-    if (this.numerator !== undefined)
```

Comment:
the undefined check should have been enough

Reply:
I have made the undefined check.

Reply:
yes, now we just have undefined check but I think I just moved the direction. You want me to revert the direction as it was earlier ?